src/validateEmail.test.js

```diff
@@ -8,13 +9,38 @@ describe(`Function 'validateEmail':`, () => {
   });
 
   it(`should return boolean`, () => {
-
+    expect(typeof validateEmail('test838@gmail.com.')).toBe('boolean');
   });
 
   it(`should return 'true' for the valid email`, () => {
-    expect(validateEmail('test838@gmail.com.'))
-      .toBeTruthy();
+    expect(validateEmail('test838@gmail.com.')).toBeTruthy();
+  });
+
+  it(`should return 'false' if email doesn't contain @`, () => {
+    expect(validateEmail('test838.mail.com')).toBeFalsy();
+  });
+
+  it(`should return 'false' for the invalid email`, () => {
+    expect(validateEmail('test838.@gmail.com')).toBeFalsy();
+  });
+
+  it(`should return 'false' if mail address is invalid`, () => {
+    expect(validateEmail('test838.@.com')).toBeFalsy();
   });
 
-  // write more tests here
+  it(`should return 'false' if mail name contain cyrillic symbols`, () => {
+    expect(validateEmail('василий@gmail.com')).toBeFalsy();
+  });
+
+  it(`should return 'true' if mail name contain one latin symbol`, () => {
+    expect(validateEmail('a@gmail.com')).toBeTruthy();
```

Comment:
The test case checks if the function returns 'true' for an email with a single Latin character before the '@' symbol. Ensure this aligns with the task requirements, as the description does not explicitly mention this case.